packages/api/src/hooks/useCtraderAccountsList.ts

```diff
@@ -34,14 +35,21 @@ const useCtraderAccountsList = () => {
                     /** The platform of the account */
                     platform: 'ctrader' as const,
                     /** Formatted display balance */
-                    formatted_balance: `${account.display_balance} ${account.currency}`,
+                    formatted_balance: displayMoney(account.balance || 0, account.currency || 'USD', {
```

Comment:
It would be better if default balance and default currency values are also a constant instead of being hard coded.

Reply:
@matin-deriv we can do that but this is only 3 places which soon gonna be 2 places how it will help to move them in a const? 🙏

Reply:
Isn't it better not to put `USD` or any currency as default?
We had an issue if there's not currency set for an account it was falling back to our default `USD` which makes things a bit confusing!

Reply:
As I discussed this with the team, `@deriv-com/utils` has a format money functionality also maybe better to use that , I can create a another PR to address it